### PR TITLE
circle shape updated

### PR DESCRIPTION
**Pull request by singlarobin**

### CODE REVIEW

File: src/components/page/styles.module.css [hunk 1]
```diff
@@ -140,7 +146,7 @@
 
 @media screen and (min-width: 1024px) {
     .pageContent {
-        overflow: hidden;
+        overflow-y: scroll;
```
Comment (DemonDaddy22) [inline src/components/page/styles.module.css]: we are preventing overflow in web-view, aren't we?

File: src/index.css [hunk 2]
```diff
@@ -177,7 +109,7 @@ code {
 
 
 /* Dark */
```
Comment (DemonDaddy22) [inline src/index.css]: let's remove these commented out values as well

**Review by DemonDaddy22 — CHANGES_REQUESTED** [review]

File: src/components/page/styles.module.css (hunk 1 above)
Comment (singlarobin) [reply src/components/page/styles.module.css]: web view  mei jo dropdown hai vo show nhi hota completely isiliye lgaya hai vo

File: src/index.css (hunk 2 above)
Comment (singlarobin) [reply src/index.css]: tere liye rakha hai -> workflow check krne ke liye

File: src/components/page/styles.module.css (hunk 1 above)
Comment (DemonDaddy22) [reply src/components/page/styles.module.css]: 768px pe bhi nhi show hora to vo height increase karde thodi, width pe lagane se kya hoga?

File: src/index.css (hunk 2 above)
Comment (DemonDaddy22) [reply src/index.css]: lint wala workflow works, maine test karliya vo, remove karde isko

Comment (singlarobin) [reply src/index.css]: okay

**Review by DemonDaddy22 — APPROVED** [review]

**Pull request merged**